core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.ts

```diff
@@ -96,40 +96,44 @@ export class StoryEditorNavbarComponent implements OnInit {
 
   discardChanges(): void {
     this.undoRedoService.clearChanges();
-    this.storyEditorStateService.loadStory(this.story.getId());
+    if (this.story !== undefined) {
+      this.storyEditorStateService.loadStory(this.story.getId());
+    }
```

Comment:
Should we throw an error when the `story` is `undefined`?

Reply:
Done.